src/assets/scripts/client/navigationLibrary/StandardRoute/StandardRouteWaypointModel.js

```diff
@@ -235,7 +235,7 @@ export default class StandardRouteWaypointModel extends BaseModel {
      * @return {array}
```

Comment:
TODO: Remove all `position` getters, add `relativePosition` getters for all objects that use either type of position model.